frontend/src/modules/SeismicIntersection/settings.tsx

```diff
@@ -0,0 +1,362 @@
+                        />
+                    </Label>
+                    <ApiStateWrapper
+                        apiResult={seismicCubeDirectoryQuery}
+                        errorComponent={"Error loading seismic directory"}
+                        loadingComponent={<CircularProgress />}
+                    >
+                        <div className="flex flex-col gap-4 overflow-y-auto">
+                            <Label text="Seismic attribute">
+                                <Select
+                                    options={seismicAttributeOptions}
+                                    value={selectedSeismicAttribute ? [selectedSeismicAttribute] : []}
+                                    size={5}
+                                    onChange={handleSeismicAttributeChange}
+                                />
+                            </Label>
+                            <Label text={TimeTypeEnumToSeismicTimeTypeStringMapping[surveyTimeType]}>
+                                <Select
+                                    options={timeOptions}
+                                    value={selectedTime ? [selectedTime] : []}
+                                    onChange={handleSeismicTimeChange}
+                                    size={8}
+                                />
+                            </Label>
+                        </div>
+                    </ApiStateWrapper>
+                </div>
+            </CollapsibleGroup>
+            <CollapsibleGroup title="Intersection Settings" expanded={false}>
+                <Label text="Extension">
+                    <Input type={"number"} value={extension} onChange={handleExtensionChange} />
+                </Label>
+                <Label text="Z-scale">
+                    <Input type={"number"} value={zScale} onChange={handleZScaleChange} />
+                </Label>
+            </CollapsibleGroup>
+        </div>
+    );
+}
+
+function fixupSyncedOrSelectedOrFirstWellbore(
+    syncedWellbore: Wellbore | null,
+    selectedWellbore: Wellbore | null,
+    legalWellbores: Wellbore[]
+): Wellbore | null {
+    const allUuids = legalWellbores.map((elm) => elm.uuid);
+    if (syncedWellbore && allUuids.includes(syncedWellbore.uuid)) {
+        return syncedWellbore;
+    }
+    if (selectedWellbore && allUuids.includes(selectedWellbore.uuid)) {
+        return selectedWellbore;
+    }
+    if (legalWellbores.length !== 0) {
+        return legalWellbores[0];
+    }
+    return null;
+}
+
+function createOptionsFromTimeOrIntervalStrings(timeOrIntervalStrings: string[]): SelectOption[] {
+    if (timeOrIntervalStrings.length == 0) {
+        return [];
+    }
+    // '2018-01-01T00:00:00.000Z--2019-07-01T00:00:00.000Z' to '2018-01-01--2019-07-01'
+    const options = timeOrIntervalStrings.map((elm) => {
+        const date = elm.replaceAll("T00:00:00.000Z", "");
```

Comment:
This won't format the string correctly. 
The seismic meta has this format currently: 2018-01-01T00:00:00.000/2019-07-01T00:00:00.000.
https://github.com/equinor/webviz/blob/main/backend/src/services/sumo_access/seismic_access.py#L29

Similar utility used in SubsurfaceMap: https://github.com/equinor/webviz/blob/main/frontend/src/modules/SubsurfaceMap/settings.tsx#L198

Might be we have different conversions flying around now. Should decide on one and put in a shared utility.

Reply:
As of now i fix it internally, but I agree that we should have a common format and create a shared utility